### ISSUE TITLE
feat: support graphics-aria and graphics-aam

### ISSUE THREAD

**Issue by jlp-craigmorten** [guidepup/virtual-screen-reader#64]
# Issue

## Details

Currently VSR does not understand elements with implicit graphics roles, resulting in them being ignored in it's accessibility tree.

This should be implemented so that VSR can simulate real screen readers as closely as possible and adhere to specification.

See:

- https://www.w3.org/TR/graphics-aria-1.0
- https://www.w3.org/TR/graphics-aam-1.0

**Comment by drwpow** [comment]
I’ve also been needing this from libraries. I see this library uses aria-query for a11y tree info. I made a faster replacement, [html-aria](https://github.com/drwpow/html-aria), that could be considered if it’s helpful. It has full ARIA 1.3 and graphics support, and parses roles according to the full ARIA spec (or if deviates in any way, please open an issue 🙂)

**Comment by drwpow-figma** [comment]
> can raise issues if needed.

Yes, please! I am actively working on this so open as many issues as you’d find. You’d be helping me out as well.

I also wouldn’t mind breaking changes to the API either if you have any ergonomic suggestions.
